_posts/2019-10-02-cockpit-204.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cockpit 204
+author: pitti
+date: 2019-10-02
+tags: cockpit linux
+slug: cockpit-204
+category: release
+summary: Cockpit with Services and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 204.
+
+### System: Highlight failed services
+
+The initial System page shows a "System Health" table entry if there are any
+failed systemd services:
+
+![Failed services on System page](/images/system-failed-services.png)
+
+The navigation menu will have a similar warning icon next to the "Services"
+entry in that case.
```

Comment:
Failed services also cause a warning to appear in the navigation, next to the "Services" entry.